velox/duckdb/conversion/DuckParser.h

```diff
@@ -86,6 +86,8 @@ struct IExprWindowFunction {
       orderBy;
 };
 
-const IExprWindowFunction parseWindowExpr(const std::string& windowString);
+const IExprWindowFunction parseWindowExpr(
+    const std::string& windowString,
+    const ParseOptions& paramParseOptions = {});
```

Comment:
Please can you describe what the defaults are for ParseOptions in a function comment.